### ISSUE TITLE
GitVersion formula has wrong url path?

### ISSUE THREAD

**Issue by BertelBB** [Homebrew/linuxbrew-core#18960]
I'm running Ubuntu on Windows subsystem and when I run GitVersion after installing with a fresh Homebrew installation, I get this error
```
/home/linuxbrew/.linuxbrew/bin/gitversion: 2: exec: /home/linuxbrew/.linuxbrew/Cellar/gitversion/5.1.3/libexec/GitVersion: Exec format error
```

Like I mentioned, it's a fresh installation of Homebrew. Still, I have tried running doctor and update with no effect.

If you look in the formula [file](https://github.com/Homebrew/linuxbrew-core/blob/master/Formula/gitversion.rb) for GitVersion you'll see that the url points to the **osx** version of GitVersion. I'm simply wondering if this is correct and if it shouldn't point to "https://github.com/GitTools/GitVersion/releases/download/5.1.3/gitversion-linux-5.1.3.tar.gz" instead?

**Comment by dawidd6** [comment]
Yes, the `url` in formula is wrong and simply changing `osx` to `linux` in it should fix the issue.

**Comment by BertelBB** [comment]
@dawidd6 I've never contributed do Homebrew before and I'm not really sure how it works.

I tried following the instructions in `CONTRIBUTING.md` to contribute a [fix](https://github.com/Homebrew/linuxbrew-core/blob/master/CONTRIBUTING.md#to-contribute-a-fix-to-the-foo-formula) to a formula. 

I updated the `gitversion.rb` formula locally and installed it with the `--build-from-source` flag. That worked fine and I got GitVersion to work, but when I run `brew test gitversion` it complains about git permissions. I'm guessing that's because then I'm changing directly to the master branch and what I should have done is fork the repo and make the changes there.

I'm just a bit confused of how the workflow is because you're mixing git and brew commands...
